ibm/mas_devops/roles/ocp_provision/defaults/main.yml

```diff
@@ -90,3 +90,5 @@ ipi_config_dir: "{{ ipi_dir }}/config/{{ cluster_name }}"
 
 ocp_installer_dir: "{{ ipi_dir }}/installer/{{ ocp_version }}"
 ocp_installer_exe: "{{ ipi_dir }}/installer/{{ ocp_version }}/openshift-install"
+
+ocp_fips: "{{ lookup('env', 'OCP_FIPS') | default('no', true) }}"
```

Comment:
Can we make this a boolean value, e.g.

```
ocp_fips: "{{ lookup('env', 'OCP_FIPS') | default('false', true) | bool }}"
```

Also need to update the README to add this new parameter to the documentation